CONTRIBUTING.md

```diff
@@ -6,6 +6,28 @@ agree to abide to the `Airy` [code of conduct](/code_of_conduct.md).
 Please refer to our [developers' manual](/docs/docs/developers-manual.md) to learn
 how to build, run, and test the `Airy Core Platform`.
 
+
+## Open an issue
+
+We track our development on Github using [issues](https://github.com/airyhq/airy/issues), so every Pull Request
+that is opened should refer to an issue.
+
+
+## Label the issue
+
+We use labels to help track the development. If possible, label your issue as soon as it is created.
+
+## Branches
+
+Feature branches must start with `feature/*`, have the issue number and a description.
+If for example the issue is named "Map message content" and it's issue number 216, a good name would be
+
+`feature/216-map-message-content`
```

Comment:
I wonder if only having `feature` will confuse people. What are we supposed to do when it's a `bug` for example?